python/setup.py

```diff
@@ -209,7 +209,8 @@ def get_packages(self):
             "gpustat >= 1.0.0b1",  # for windows
             "opencensus",
             "prometheus_client >= 0.7.1",
-            "smart_open"
+            "smart_open",
+            "virtualenv",  # For pip runtime env.
```

Comment:
@rkooo567 is currently making it so runtime_env is supported in the basic installation rather than `ray[default]`, so we'd want to put it there instead.

Reply:
I moved `virtualenv` to the `setup_spec.install_requires` in setup.py